common/common-rest/src/test/java/io/servicecomb/common/rest/TestAbstractRestInvocation.java

```diff
@@ -202,6 +202,7 @@ public void invokeFilterHaveResponse(@Mocked HttpServerFilter filter) {
     restInvocation = new AbstractRestInvocation() {
       @Override
       protected void doInvoke() throws Throwable {
+        result.value = Response.ok("not run to here");
```

Comment:
how about change to "This should not run" or "expect not to run here"?

Reply:
i feet that this is not necessary